### ISSUE TITLE
Splitting GF into separate repositories

### ISSUE THREAD

**Issue by johnjcamilleri** [GrammaticalFramework/GF#47]
There is an ongoing discussion about this issue [on the gf-dev mailing list](https://groups.google.com/d/topic/gf-dev/fedRMIi44pE/discussion).

**Comment by fredefox** [comment]
I think this is a great idea. Particularly I'm interested in splitting `pgf` the Haskell bindings into a seperate repo. I imagine I'd just make a seperate repo for that and add it as a git submodule in the same location here. The build instructions will change slightly, however, let me know if this is something you are interested in.

**Comment by krangelov** [comment]
I am very much against splitting the runtimes from the compiler. Changes in
the compiler are almost always linked to changes in the runtime and vice
versa. Having them in different repositories is sure receipe for getting
inconsitencies. It will also mean that for every change, instead of one
patch I will have to make two.

**Comment by johnjcamilleri** [comment]
I agree with Krasimir here. Things which are closely coupled should remain in the same repository.
